### PR TITLE
Models

### PR DESCRIPTION
**Pull request by mBerezovskyy**

### CODE REVIEW

**Review by Ravenen — CHANGES_REQUESTED** [review]
library_item i book - one-to-one
Створити табличку library_item і табличку book. id book'a - foreign key до id library_item.
В моделі book створити поле library_item, яке буде library_item по id.

[Mapping Inheritance to Relational Databases](https://aksakalli.github.io/2015/06/16/mapping-inheritance-to-relational-databases.html)

Пройтись по полях, пофіксити зв'язки.

File: models/library_item.py [hunk 1]
```diff
@@ -11,3 +12,7 @@ class LibraryItem(utils.db.Base):
 
     def __repr__(self):
         return str(self.__dict__)
+
+    def calculate_rent_price_per_day(self) -> float:
+        result = self.pledgePrice / self.quantity
+        return percentage_from(number=result, percent=25)
```
Comment (Ravenen) [inline models/library_item.py]: It's better to move the percentage value into constant.

File: utils/helpers.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+def percentage_from(number: float, percent: int) -> float:
+    return (number * percent) / 100
```
Comment (Ravenen) [inline utils/helpers.py]: As this function works with money, better make it work with Decimal instead of float.

**Review by Ravenen — APPROVED** [review]

File: models/library_item.py (hunk 1 above)
Comment (KhrystynaVysotska) [reply models/library_item.py]: Resolved

File: utils/helpers.py (hunk 2 above)
Comment (KhrystynaVysotska) [reply utils/helpers.py]: Resolved

**Review by Ravenen — APPROVED** [review]

**Review by Ari100telll — APPROVED** [review]

**Pull request merged**